server/src/termdb.violin.js

```diff
@@ -167,7 +167,7 @@ function divideValues(q, data, term, overlayTerm) {
 
 	for (const [c, v] of Object.entries(data.samples)) {
 		//if there is no value for term then skip that.
-		const value = roundValue(v[term.id]?.value, 1)
+		const value = roundValue(v[term.id]?.value, 3)
```

Comment:
Do not round at all. Different data types will have different numbers of decimal places. Raw data should be plotted.